client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -117,17 +112,8 @@ const PluginDetailsCTA = ( { plugin, isPlaceholder } ) => {
 	}, [ selectedSite?.slug ] );
 
 	const getSaasRedirectHRef = useCallback( () => {
```

Comment:
Not added in this change, but I'm wondering why this is a callback if we only use its value based on properties. Maybe a `useMemo` would better fit here.

Reference: https://reactjs.org/docs/hooks-reference.html#usememo